src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue/rebuild.groovy

```diff
@@ -49,7 +50,8 @@ drawParameterBody(parameterDefinition, f, valueIt, axes, project, build, layoute
 
 
 private void drawParameterBody(MatrixCombinationsParameterDefinition paramDef, Namespace f,valueIt,AxisList axes,MatrixProject project,MatrixBuild build,Layouter layouter) {
-    f.entry(title: valueIt.getName(), description: it.getDescription()) {
+    set("escapeEntryTitleAndDescription", false);
+    f.entry(title: h.escape(valueIt.name), description: it.formattedDescription) {
```

Comment:
`f.entry` of Jenkins < 2.32.2 outputs values as-is.
`f.entry` of Jenkins >= 2.32.2 outputs values as-is when `escapeEntryTitleAndDescription` is `false` (`true` is default).